Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -2864,171 +3212,453 @@ def extract_sentinelone_zip_file(zip_file_data: bytes, password: str) -> Tuple[s
 
     file_names = [name for name in zip_file.namelist() if name != "manifest.json"]
     if len(file_names) < 1:
-        raise DemistoException("No file found in upload from agent. Perhaps the path submitted is wrong?")
+        raise DemistoException(
+            "No file found in upload from agent. Perhaps the path submitted is wrong?"
+        )
 
     file_name = file_names[0]
-    file_data = zip_file.read(file_name, password.encode('utf-8'))
+    file_data = zip_file.read(file_name, password.encode("utf-8"))
     return file_name, file_data
 
 
 def download_fetched_file(client: Client, args: dict) -> List[CommandResults]:
     """
     Download a file that has been requested by `fetch-file`
     """
-    agent_id = args.get('agent_id')
-    activity_id = args.get('activity_id')
-    password = args.get('password')
+    agent_id = args.get("agent_id")
+    activity_id = args.get("activity_id")
+    password = args.get("password")
     assert isinstance(password, str)
 
     zip_file_data = client.download_fetched_file_request(agent_id, activity_id)
     path, file_data = extract_sentinelone_zip_file(zip_file_data, password)
-    return [CommandResults(readable_output=f"Successfully downloaded file `{path}`",
-                           outputs_prefix='SentinelOne.Download',
-                           outputs_key_field='Path',
-                           outputs={'Path': path}),
-            fileResult(f"{path.replace('/', '_')}", file_data)]
+    return [
+        CommandResults(
+            readable_output=f"Successfully downloaded file `{path}`",
+            outputs_prefix="SentinelOne.Download",
+            outputs_key_field="Path",
+            outputs={"Path": path},
+        ),
+        fileResult(f"{path.replace('/', '_')}", file_data),
+    ]
+
+
+def run_remote_script_command(client: Client, args: dict) -> CommandResults:
+    """
+    Run a remote script that was uploaded to the SentinelOne Script Library
+    """
+
+    context = {}
+    # Get arguments
+    account_ids = argToList(args.get("account_ids"))
+    script_id = args.get("script_id")
+    output_destination = args.get("output_destination")
+    task_description = args.get("task_description")
+    output_directory = args.get("output_directory")
+    agent_ids = argToList(args.get("agent_ids"))
+
+    run_remote_script = client.run_remote_script_request(
+        account_ids, script_id, output_destination, task_description, output_directory, agent_ids)
+
+    if run_remote_script:
+        context = {
+            "pendingExecutionId": run_remote_script.get("pendingExecutionId"),
+            "pending": run_remote_script.get("pending"),
+            "affected": run_remote_script.get("affected"),
+            "parentTaskId": run_remote_script.get("parentTaskId")
+        }
```

Comment:
This is redundant - you can send `run_remote_script` directly, with the wanted header to `tableToMarkdown` under the `headers` argument.